typescript/infra/config/environments/mainnet3/agent.ts

```diff
@@ -501,7 +550,7 @@ const releaseCandidate: RootAgentConfig = {
     rpcConsensusType: RpcConsensusType.Fallback,
     docker: {
       repo,
-      tag: '4cb2c9a-20241205-142854',
+      tag: 'b98678c-20241213-043725',
```

Comment:
you can see this running in RC atm https://solscan.io/account/G6fWKEBSrDtR8Uzx4kdVRRhrsHZ6zmGX1G2TYMmMgK6u - though might run out of funds. I'd expect some reverts as it's competing with the hyperlane relayer

check out the fees!